### PR TITLE
Add notifications guide

### PR DESCRIPTION
**Pull request by stefanprodan**

### CODE REVIEW

File: docs/guides/notifications.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Setup Notifications
+
+When operating a cluster, different teams may wish to receive notification about
```
Comment (hiddeco) [inline docs/guides/notifications.md:3]:
```suggestion
When operating a cluster, different teams may wish to receive notifications about
```

File: docs/guides/notifications.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      name: webapp-backend
+    - kind: Kustomization
+      name: webapp-frontend
+```
+
+Apply the above files or commit them to the `fleet-infra` repository.
+
+To verify that the alert has been acknowledge by the notification controller do:
+
+```text
```
Comment (hiddeco) [inline docs/guides/notifications.md:72]:
```suggestion
```console
```

Comment (stefanprodan) [reply docs/guides/notifications.md:72]: console is not suitable for commands and outputs in the same block, I used text like in the get-started guide.

Comment (hiddeco) [reply docs/guides/notifications.md:72]: Console _is meant to be used_ for command and outputs in the same block. See:

```console
$ kubectl -n gitops-system get alerts
NAME             READY   STATUS        AGE
on-call-webapp   True    Initialized   1m
$ kubectl -n gitops-system get alerts
NAME             READY   STATUS        AGE
on-call-webapp   True    Initialized   1m
```

**Pull request merged**